ipyparallel/client/client.py

```diff
@@ -161,11 +162,30 @@ def _plaintext(self) -> str:
         if not text_out:
             return ''
 
+        ip = get_ipython()
+        if ip is None:
+            colors = "NoColor"
+        else:
+            colors = ip.colors
+
+        if colors == "NoColor":
```

Comment:
```suggestion
        if colors in ("NoColor", 'nocolor'):
```

IPython 9.0 supports lowercase.

Reply:
thanks!